pattern-filter/apps/pattern-filter/src/main/java/com/redhat/training/transform/TransformRouteBuilder.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.training.transform;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.stereotype.Component;
+import org.springframework.context.annotation.Bean;
+import org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory;
+import org.apache.camel.component.jms.JmsComponent;
+import javax.jms.JMSException;
+
+//import org.apache.camel.model.dataformat.XmlJsonDataFormat;
+
+@Component
+public class TransformRouteBuilder extends RouteBuilder {
+
+    //TODO add the XmlJsonDataFormat
+
+    @Override
+    public void configure() throws Exception {
+
+        from("jms:queue:orderInput")
+            .routeId("Transforming Orders")
+            .marshal().jaxb()
+            .log("XML Body: ${body}")
+            .to("mock:fufillmentSystem");
+
+        //TODO add direct route to mock order log end point
+    }
+
+    @Bean
+    public JmsComponent jmsComponent() throws JMSException {
+        // Create the connectionfactory which will be used to connect to Artemis
```

Comment:
I think that this kind of comments are confusing, the main reason is because we are using an "imperative" sentence, so it loos like I, as student, need to do something. Perhaps we can tweak them a little bit and transform then into a sentence that describes what the code does, for example here: `Creates the connection...`.

This comment is for the whole PR. Let's try to keep "imperative" sentences for the "TODO" comments, and more descriptive sentences for any other type of comment in the code.